packages/volto/src/components/manage/Blocks/Listing/SummaryTemplate.jsx

```diff
@@ -22,17 +22,28 @@ const SummaryTemplate = ({ items, linkTitle, linkHref, isEditMode }) => {
   return (
     <>
       <div className="items">
-        {items.map((item) => (
-          <div className="listing-item" key={item['@id']}>
-            <ConditionalLink item={item} condition={!isEditMode}>
-              <Component componentName="PreviewImage" item={item} alt="" />
-              <div className="listing-body">
-                <h3>{item.title || item.id}</h3>
-                <p>{item.description}</p>
+        <ul className="ui list summary-listing">
+          {items.map((item) => (
+            <li key={item['@id']}>
+              <div
+                className={`listing-item ${
+                  item.review_state ? `state-${item.review_state}` : ''
+                }`}
+                role="listitem"
```

Comment:
The `role="listitem"` isn't needed here as the containing `<li>` element already provides this role
```suggestion
```